generic.go

```diff
@@ -179,6 +179,14 @@ func Delete[V any](slice *[]V, i int, optionalJ ...int) error {
 	return nil
 }
 
+func Tally[T comparable](slice *[]T) map[T]int {
```

Comment:
Any particular reason to take a pointer to a slice here?

Reply:
copy&pasta 😅